src/bitcoind/mod.rs

```diff
@@ -23,6 +23,10 @@ use jsonrpc::{
     simple_http,
 };
 
+/// Number of try the client is allowed to do in case of timeout and i/o error
+/// while communicating with the bitcoin daemon.
+const BITCOIND_TRY_LIMIT: usize = 6;
```

Comment:
That a huge reduction in the number of retries. What is the rationale here? We would previously support spurious failures of `bitcoind`  for up to 1min. After this patch we'd fail after `6 * .5 = 3` seconds.